src/pages/about.js

```diff
@@ -1,66 +1,118 @@
+/* eslint-disable react/prop-types */
 import React from 'react';
-
+import { graphql } from 'gatsby';
 import Layout from '../components/layout';
 import SEO from '../components/seo';
+import Img from 'gatsby-image';
+import LearnTeachCoding from '../assets/Learn-Teach-Coding.svg';
+
+function AboutPage(props) {
+  // const { avatarSethAlexanderImage } = [
+  //   data.avatarSethAlexanderImage.childImageSharp.fluid,
+  // ];
+  console.log(props.data);
 
-function AboutPage() {
   return (
     <Layout>
-      <SEO keywords={[`nashville`, `freecodecamp`]} title="Home" />
-
-      <section className="text-center">
-        <h2 className="inline-block p-3 mb-4 text-2xl font-bold bg-yellow-400">
-          Code of Conduct
-        </h2>
+      <SEO keywords={[`nashville`, `freecodecamp`]} title="About" />
+      <section className="main-section md:bg-FCCblue-100">
+        <div className="main-section-content main-section-index-1">
+          <div className="pt-8 px-2 md:px-0 md:mr-8 md:w-8/12">
+            <h2 className=" text-FCCblue-200">
+              <span>ABOUT freeCodeCamp Nashville</span>
+            </h2>
 
-        <p className="text-left leading-loose">
-          In the interest of fostering an open and welcoming environment, we
-          pledge to making participation in our community a harassment-free
-          experience for everyone, regardless of age, body size, disability,
-          ethnicity, gender identity and expression, level of experience,
-          nationality, personal appearance, race, religion, or sexual identity
-          and orientation.
-        </p>
+            <p className="mb-5">
```

Comment:
This is cool. I learned tailwind uses a 1:4 size ratio with rem units after searching for what mb-5 means.